app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -2,37 +2,43 @@
   <fieldset ng-disabled="claimDisabled">
     <div ng-show="storageClasses | hashSize" class="form-group">
       <!--storage class-->
-      <label class="required">Storage Classes</label>
+      <label>Storage Class</label>
+      <div>
+      <ui-select
+        ng-if="storageClasses"
+        ng-model="claim.storageClass"
+        theme="bootstrap"
+        search-enabled="true"
+        title="Select a storage class"
+        class="select-role"
+        flex>
+        <ui-select-match placeholder="Select a storage class">
+          <span>
+            {{$select.selected.metadata.name}}
+          </span>
+        </ui-select-match>
+        <ui-select-choices
+          repeat="sclass as sclass in storageClasses | toArray | filter : { metadata: { name: $select.search } } | orderBy: 'metadata.name'">
+          <div>{{ sclass.metadata.name }}
```

Comment:
Sorry, @rhamilto I'm wrong. The problem is a typo because the span is not closed.

```html
<span ng-bind-html="sclass.metadata.name | highlight : $select.search"></span>
```